.github/workflows/lektor-check.yml

```diff
@@ -0,0 +1,39 @@
+name: lektor check
+on: [push, pull_request]
```

Comment:
I'd probably only do this for pull-requests as branches could be quite "unready" and we already had some issues with running into quota limitations.

Reply:
Welche Issues hattest du denn?????

Reply:
Die FFBsee orga hat nachwievor Probleme innerhalb des LFS Quota zu bleiben. Ich hab gesehen dass du versucht hast das dadurch zu lösen dass die Daten zu dem gitea bei see-base ausgelagert werden, aber das ist auch nur ein Workaround, da dann jeder einen Account bei see-base braucht und das Repo nun von zwei Servern abhängig ist.

Ich hab noch nicht nachgeschaut wie lange die Änderung mit LFS schon her ist, aber momentan ist das Quota noch stark überzogen.

Reply:
Da darfst du lange warten.

Das LFS Quota wird so schnell auch nicht wieder frei werden bis microsoft das umstellt!

Reply:
Dann schau nochmal nach und dann wirst du selber feststellen das diese aussage inhaltlich nicht ganz zutreffend ist!